code/modules/mob/living/carbon/examine.dm

```diff
@@ -163,7 +163,7 @@
 	for(var/obj/item/bodypart/part as anything in bodyparts)
 		if(part.body_zone in covered_zones)
 			continue
-		if(part.limb_id != (dna.species.examine_limb_id ? dna.species.examine_limb_id : dna.species.id))
+		if(dna && (part.limb_id != (dna.species.examine_limb_id ? dna.species.examine_limb_id : dna.species.id)))
```

Comment:
Can we check for dna BEFORE we start checking limbs? Because without DNA, this entire bodypart checking is useless with this code.